Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.service.opg.digital/opguk/php-fpm
+FROM registry.service.opg.digital/opguk/php-fpm:0.1.207
```

Comment:
@brettminnie  I suggested that after we had a disruption due to a non-working version. If there is an update on php-fpm I prefer to change the Dockerfile in a branch and test it there, unless you guarantee php-fpm latest version is always stable